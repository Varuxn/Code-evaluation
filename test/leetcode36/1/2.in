10
1 1 1 -2 -4 -5 6 -5 -3 0 