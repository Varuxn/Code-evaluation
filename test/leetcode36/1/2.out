-5 -5 
-5 -5 -3 
-5 -5 -3 0 
-5 -5 0 
-5 -3 
-5 -3 0 
-5 0 
-5 6 
-4 -3 
-4 -3 0 
-4 0 
-4 6 
-3 0 
-2 0 
-2 6 
1 1 
1 1 1 
1 1 1 6 
1 1 6 
1 6 
